src/views/Scenario/Scenario.js

```diff
@@ -28,8 +28,26 @@ import { AppInsights } from '../../services/AppInsights';
 import { PERMISSIONS } from '../../services/config/Permissions';
 import { PermissionsGate } from '../../components/PermissionsGate';
 import { getCreateScenarioDialogLabels, getReportLabels } from './labels';
-import useStyles from './style';
+// import useStyles from './style';
 import { useNavigate, useParams } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    paddingTop: '16px',
+    paddingLeft: '8px',
+    paddingRight: '8px',
+  },
+  rightButtons: {
```

Comment:
can we rename this class to "rightButton" because we are applying it on each button individually not both together ? :slightly_smiling_face:

Reply:
Sure